test/test_energy_storage_device.py

```diff
@@ -322,6 +325,107 @@ def test_accuracy_pycap_simulation(self):
                                         output['voltage'][i-1])
             # update index
             i = i + 1
+    
+    def test_compatibilty_other_dualfoil(self):
+        # PURPOSE: test to see if Dualfoil object works with the
+        #   version of dualfoil without restard (5.2)
+        df = Dualfoil(path=path2, input_name='li-ion.in',
+                      restart_capable=False)
+
+        # Go through evolve tests. Affirm the following:
+        #   1. Variable held constant stays constant
+        #   2. Dependent variable changes in an expected way
```

Comment:
This test is pretty weak. It doesn't check any value. If you do a constant voltage and the current changes from -10 to -1000 you won't see it.